puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
"Anytime you have to destroy and rebuild it's inconvenient and time consuming." I'm amused because that is typically @beporter's line more or less.